src/fmu/dataio/__init__.py

```diff
@@ -2,8 +2,8 @@
 
 from fmu.dataio.dataio import (
     AggregatedData,
+    CreateCaseMetada,
```

Comment:
```suggestion
    CreateCaseMetadata,
```
missed the last two characters here 😄 will need to be adjusted everywhere